scripts/extensions/pool_update.precheck

```diff
@@ -75,7 +75,9 @@ def parse_control_package(session, yum_url):
 def execute_precheck(session, control_package, yum_conf_file):
     if not control_package:
         return 'ok'
-    livepatch_dic = {'PATCH_PRECHECK_LIVEPATCH_COMPLETE': 'ok_livepatch_complete', 'PATCH_PRECHECK_LIVEPATCH_INCOMPLETE': 'ok_livepatch_incomplete', 'PATCH_PRECHECK_LIVEPATCH_NOT_APPLICABLE': 'ok'}
+    livepatch_dic = {'PATCH_PRECHECK_LIVEPATCH_COMPLETE': 'ok_livepatch_complete',
```

Comment:
I think the name `livepatch_dic` could be improved: `dict` would be more idiomatic that `dic` but in general I think types should be not encoded in names of values. Also `livepatch` does not convey what what this dictionary holds. As I understand it, the dictionary holds messages from Yum that are mapped to strings. I'd call it `yum_messages` or similar. When traversing the dictionary below, I likewise would not use `key` but `msg` - the code is looking for a Yum message.